be/docker/be-dev.dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:lts-alpine
+
+WORKDIR /app
+
+# append node_modules to PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# copy project files (docker-compose seems ignore '.dockerignore' files.
+# If you can make docker-compose ignore folders such as 'node_modules' please
+# make a pull request doing that (and replace these copy commands with 'COPY . ./').)
```

Comment:
Will create an issue about this, let's deal with it in a separate pr

Reply:
Or you know what, create the issue yourself and assign to yourself/Sten

Reply:
Please link the issue here when it gets created.

Reply:
Created issue #45.